### DISCUSSION TITLE
[Feature request]:Add feature of application not grouping.

### DISCUSSION THREAD

**Discussion by MNWZ211** [valinet/ExplorerPatcher#3406] [Ideas]
If there are repeated discussions, please ignore them. There are too many discussions and it is difficult to find them.

**Comment by pyrates999** [comment]
You can do this with windhawk: https://windhawk.net/mods/taskbar-grouping

**Reply by MNWZ211** [reply]
ok
thanks

**Reply by mustafatoktas** [reply]
> You can do this with windhawk: https://windhawk.net/mods/taskbar-grouping

@pyrates999  , I didn't like you very much because you always wrote the same thing to everyone (like update to the pre-release version, write the windows version, etc.), but I discovered windhawk thanks to you. There are many mods that work for me. Thank you.
